components/brave_rewards/resources/extension/brave_rewards/utils.ts

```diff
@@ -48,8 +49,19 @@ export const getGrants = (grants?: RewardsExtension.Grant[]) => {
 }
 
 export const getGrant = (grant?: RewardsExtension.GrantInfo) => {
-  if (grant && grant.type === 'ads') {
+  if (!grant) {
+    return grant
+  }
+
+  grant.finishTitle = getMessage('grantFinishTitleUGP')
+  grant.finishText = getMessage('grantFinishTextUGP')
+  grant.finishTokenTitle = getMessage('grantFinishTokenTitleUGP')
+
+  if (grant.type === 'ads') {
     grant.expiryTime = 0
+    grant.finishText = getMessage('grantFinishTextAds')
```

Comment:
nit: for consistency swap text/title lines around so they match above

Reply:
done